examples/integrations/yolo/import_yolo_annotations.ipynb

```diff
@@ -0,0 +1,312 @@
+{
```

Comment:
<p>Initialize our model for image data rows using <code>yolov8n-seg.pt</code>, which supports segmentation masks. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1673/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Done